src/internal/m365/collection/drive/item.go

```diff
@@ -121,6 +125,19 @@ func downloadFile(
 		return nil, clues.New("empty file url").WithClues(ctx)
 	}
 
+	// Precheck for url expiry before we make a call to graph to download the
+	// file. If the url is expired, we can return early and save a call to graph.
+	//
+	// Ignore all errors encountered during the check. We can rely on graph to
+	// return errors on malformed urls. Ignoring errors also future proofs against
+	// any sudden graph changes, for e.g. if graph decides to emb the token under a
```

Comment:
nit: not sure if `emb` is a typo or a term I'm unfamiliar with. Could you clarify it in the comment please?

Reply:
thanks! it's a typo